src/plugins/intel_cpu/src/nodes/kernels/x64/random_uniform.cpp

```diff
@@ -4,29 +4,64 @@
 
 #include "random_uniform.hpp"
 
+#include <immintrin.h>
+
 using namespace dnnl::impl::cpu;
 
 namespace ov {
 namespace intel_cpu {
 namespace kernel {
+namespace random_uniform {
+
+#define GET_PHILOX_OFFSET(field) offsetof(PhiloxGeneratorCallArgs, field)
+
+#define GET_MERSENNE_OFFSET(field) offsetof(MersenneTwisterGeneratorCallArgs, field)
+
+#define BROADCAST_CONSTANT(func, vector, aux_register, constant)                            \
+    mov(aux_register, constant);                                                            \
+    func(vector, aux_register);
+
+#define BROADCAST_PHILOX_PARAM(func, vector, aux_register, param_name)                      \
+    mov(aux_register, ptr[r64_params + GET_PHILOX_OFFSET(param_name)]);                     \
+    func(vector, ptr[aux_register]);
+
+#define BROADCAST_MERSENNE_PARAM(func, vector, aux_register, param_name)                    \
+    mov(aux_register, ptr[r64_params + GET_MERSENNE_OFFSET(param_name)]);                   \
+    func(vector, ptr[aux_register]);
+
+#define INIT_1_ELEM_T_VAL(A, V, R, T)                                                       \
+    static const T A[1] = { V };                                                            \
+    mov(R, reinterpret_cast<uintptr_t>(A));                                                 \
+
+#define INIT_8_ELEM_T_ARR(A, V, R, T)                                                       \
```

Comment:
Actually it initializes 4 elements for SSE.

Reply:
Ah, true. Reverted to INIT_ARR